CMakeLists.txt

```diff
@@ -113,6 +113,7 @@ add_library(lcf
 	src/generated/lsd_savevehiclelocation.cpp
 	src/generated/rpg_chipset.cpp
 	src/generated/rpg_mapinfo.cpp
+	src/generated/rpg_system.cpp
```

Comment:
Please add this also to [Makefile.am](https://github.com/EasyRPG/liblcf/blob/master/Makefile.am#L26). This will fix the test failure as well.

Reply:
Added